spec/features/create_new_collection_spec.rb

```diff
@@ -43,14 +43,22 @@
       expect(page).to have_content('Depositing')
 
       # Simulate the deposit process
-      CollectionVersion.last.tap do |version|
-        version.collection.update!(druid: 'druid:dc224fz4940')
-        version.deposit_complete!
-      end
+      collection_version = CollectionVersion.last
```

Comment:
i'm now making use of this object in other expectations below, so I am saving it here